test/jdk/java/time/test/java/time/TestInstant.java

```diff
@@ -96,4 +98,11 @@ public void test_epochMillis(String name, long millis) {
         assertEquals(millis, m, name);
     }
 
+    @Test
+    public void test_microsUntil() {
```

Comment:
A comment about the test might be helpful.

Reply:
Indeed. Added some comments to the test.